io/src/main/java/com/github/jorgecastillo/kotlinandroid/io/runtime/ui/SuperHeroDetailActivity.kt

```diff
@@ -17,50 +18,51 @@ import kotlinx.android.synthetic.main.activity_detail.*
 
 class SuperHeroDetailActivity : AppCompatActivity(), SuperHeroDetailView {
 
-  companion object {
-    const val EXTRA_HERO_ID = "EXTRA_HERO_ID"
+    companion object {
```

Comment:
2spacesverywhere!!

Reply:
No clue on what code style I'm using here :D didn't care much tbh since all are good to me as long as all people in the project share them, and I'm the only one working on this, lolz.